libraries/src/User/User.php

```diff
@@ -542,7 +542,7 @@ public function setLastVisit($timestamp = null)
      */
     public function getTimezone()
     {
-        $timezone = $this->getParam('timezone', Factory::getApplication()->get('offset', 'GMT'));
+        $timezone = $this->getParam('timezone', Factory::getApplication()->get('offset', 'UTC'));
```

Comment:
```suggestion
        $timezone = $this->getParam('timezone', Factory::getApplication()->get('offset', 'GMT'));
```
This change will be done with #43912 against 6.0.